### PR TITLE
Set-up server with express

### PR DESCRIPTION
**Pull request by PrinceOm**

### CODE REVIEW

File: config/environment/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const _ = require('lodash');
+
+const baseSettings = {
+  env: process.env.NODE_ENV,
+  port: process.env.PORT || 3000,
```
Comment (dting) [inline config/environment/index.js]: Lets just change this to default to `8888` too.

File: config/environment/development.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  port: process.env.PORT || 8888,
```
Comment (dting) [inline config/environment/development.js]: Let's let this fall through to the `index.js` default.

File: config/express.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const parser = require('body-parser');
+const morgan = require('morgan');
+const environment = require('./environment/development');
```
Comment (dting) [inline config/express.js]: I think this should be `require('./environment')`.

**Comment by PrinceOm** [comment]
done

**Comment by dting** [comment]
LGTM

File: config/environment/index.js (hunk 1 above)
Comment (PrinceOm) [inline config/environment/index.js]: Done.

File: config/environment/development.js (hunk 2 above)
Comment (PrinceOm) [inline config/environment/development.js]: Done.

**Pull request merged**